src/components/features/components/cities.tsx

```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { FiNavigation } from 'react-icons/fi';
+import { ReactElement, useMemo } from 'react';
+
+export const cityLocations = [
+  { city: 'Москва', branch: 92 },
+  { city: 'Санкт-Петербург', branch: 16 },
+  { city: 'Калининград', branch: 10 },
+  { city: 'Красногорск', branch: 5 },
+  { city: 'Воронеж', branch: 4 },
+  { city: 'Долгопрудный', branch: 3 },
+  { city: 'Красноярск', branch: 3 },
+  { city: 'Раменское', branch: 3 },
+];
+
+export function Location(): ReactElement {
```

Comment:
one more time: this component name and file name should match. Currently they do not: `cities.tsx` and `Location`. Please, select one and fix